### ISSUE TITLE
Deprecation of methods as being dropped in 2.x 

### ISSUE THREAD

**Issue by raziel057** [doctrine/inflector#154]
It's strange that you deprecate the following methods (tableize, classify, ...), because it continue to be used in Doctrine ORM: 

For example ``Inflector::classify`` is used in:
https://github.com/doctrine/orm/blob/master/lib/Doctrine/ORM/EntityRepository.php

**Comment by alcaeus** [comment]
The deprecation refers to the legacy API and instructs users to switch to the new API (found in the Doctrine\Inflector namespace without “Common”). The logic is still available.

**Comment by Healyhatman** [comment]
> ... instructs users to switch to the new API (found in the Doctrine\Inflector namespace without “Common”)

On Laravel what would I replace this with when adding new rules? Because Doctrine\Inflector\Inflector::rules is no good
```
Doctrine\Common\Inflector\Inflector::rules('plural', [
    '/^is$/' => 'are',
    '/^Is$/' => 'Are'
]);
```

**Comment by ninjasitm** [comment]
@Healyhatman I'm also experiencing the same issue but there is no documentation for how to migrate from this.

@alcaeus Do you have any insight on how to migrate to the new method of adding irregular inflection rules?

**Comment by garrettw** [comment]
@Healyhatman @ninjasitm There is already documentation written for this: https://github.com/doctrine/inflector/blob/master/docs/en/index.rst

**Comment by ninjasitm** [comment]
@garrettw Yes. But how do we do this in Laravel? Here you have to create an inflector instance where as before it was created statically. Can you help me out on this in case I'm missing something really simple?

**Comment by malarzm** [comment]
@ninjasitm I'd suggest asking Laravel community about this. We're not documenting how other frameworks use and expose usage of Inflector to their users

**Comment by ninjasitm** [comment]
OK. Will create an issue on the `laravel/framework` github repo. If I find a solution I'll link back here.
